source/deploy/postgres-migration.rst

```diff
@@ -109,6 +109,29 @@ To drop indexes, run the following commands before the migration (These are incl
    DROP INDEX IF EXISTS idx_posts_message_txt;
    DROP INDEX IF EXISTS idx_fileinfo_content_txt;
 
+Artifacts may remaining from previous configurations/versions
+~~~~~~~~~~~~~~~~~
+
+Prior to ``v6.4``, Mattermost was utilizing `golang-migrate <https://github.com/golang-migrate/migrate>`__ for handling the schema migrations. Since we don't use it anymore, the table ``schema_migrations`` shouldn't be migrated. If you were using Mattermost before ``v6.4`` consider excluding this table from migration and comparison. 
+
+.. code:: sql
+
+   DROP TABLE mattermost.schema_migrations;
```

Comment:
The "consider excluding this table from migration" sounds a bit weird, since what they need to do is drop it at the source database, right? Maybe something more explicit like "consider dropping that table before migrating"?

Reply:
Yeah, I agree.  This is actually a kind of FYI, it is already being ignored while executing the migration.

Reply:
Yeah, that's why I thought that "consider excluding this table from migration" is actually not 100% correct, since it'll be already excluded, right? And if it is being excluded, do we really need this?